CHANGELOG.rst

```diff
@@ -2,6 +2,23 @@
 Changelog
 =========
 
+0.19.0 (YYYY-MM-DD)
+-------------------
+
+* Now requires ``andi >= 0.6.0``.
+
+* Change the implementation of resolving and building item dependencies from
+  page objects. Now the ``andi`` custom builders are used to create a single
+  plan that includes building page objects and items. This fixes problems
+  such as providers being called multiple times.
+
+  * :class:`~scrapy_poet.page_input_providers.ItemProvider` is now no-op. It's
+    no longer enabled by default and users should also stop enabling it.
+  * ``PageObjectInputProvider.allow_prev_instances`` and code related to it
+    were removed so custom providers may need updating.
+
+* Fix some tests.
```

Comment:
```suggestion
* Fixed some tests.
```

Not 100% sure, it feels to me like the current form if right for commit messages but not for a changelog entry.

Reply:
We need a uniform changelog style guide :)